gpslogger/src/main/java/com/mendhak/gpslogger/GpsMainActivity.java

```diff
@@ -300,22 +449,8 @@ protected void onResume() {
 
         populateProfilesList();
         enableDisableMenuItems();
-
-
     }
 
-    @Override
-    protected void onPause() {
-        stopAndUnbindServiceIfRequired();
-        super.onPause();
-    }
-
-    protected void onStop() {
-        super.onStop();
-        if (!isFinishing()) {
-            stopAndUnbindServiceIfRequired();
-        }
-    }
```

Comment:
These weren't necessary either. The service killing gets handled by normal back and exit gestures.